plugins/GSdx/Renderers/DX11/GSDevice11.cpp

```diff
@@ -479,7 +479,28 @@ void GSDevice11::DrawPrimitive()
 
 void GSDevice11::DrawIndexedPrimitive()
 {
-	m_ctx->DrawIndexed(m_index.count, m_index.start, m_vertex.start);
+	// DX can't read from the FB
+	// So let's copy it and set that as the shader
+	// resource for slot 4 to avoid issues.
+	if (m_state.ps_sr_texture[4] && m_state.ps_sr_texture[4]->Equal(m_state.rt_texture))
+	{
+#ifdef _DEBUG
+		fprintf(stdout, "FB read detected on slot 4: copying fb...\n");
+#endif
```

Comment:
Should we make an exception here? @gregory38 asked me to print a log and I assumed he did so because we are doing something not so obvious here that might be nice for a dev to know.